SU2_CFD/src/transfer_physics.cpp

```diff
@@ -83,7 +83,7 @@ void CTransfer_FlowTraction::GetPhysical_Constants(CSolver *flow_solution, CSolv
   /*--- We have to clear the traction before applying it, because we are "adding" to node and not "setting" ---*/
 
   for (unsigned long iPoint = 0; iPoint < struct_geometry->GetnPoint(); iPoint++) 
-    struct_solution->node[iPoint]->Clear_FlowTraction();
+    struct_solution->node->Clear_FlowTraction(iPoint);
```

Comment:
There are not so many changes here, so I will work on the names of the split files in #769 so we can merge it in soonish, and I then can help updating this.